.circleci/config.yml

```diff
@@ -241,6 +247,8 @@ jobs:
             JEST_JUNIT_OUTPUT_DIR: ./reports/
       - store_test_results:
           path: apps/mark-scan/backend/reports/
+      - store_artifacts:
+          path: apps/mark-scan/backend/src/__image_snapshots__/__diff_output__/
```

Comment:
Thank you for adding this!

`__image_snapshots__` directories may appear in other places like `apps/mark-scan/backend/src/reports/__image_snapshots__/__diff_output__/`. They appear in the same directory as the test. I just did a little research and it looks like this config doesn't support a glob pattern, so I think what we'll want to do is manually add paths as they come up or change the snapshot config. 

In either case, I don't think you need to overthink or block on that. Thank you for adding this across apps.